xbmc/interfaces/json-rpc/schema/methods.json

```diff
@@ -1335,7 +1335,7 @@
       { "name": "rating", "$ref": "Optional.Number" },
       { "name": "mpaa", "$ref": "Optional.String" },
       { "name": "imdbnumber", "$ref": "Optional.String" },
-      { "name": "votes", "$ref": "Optional.String" },
+      { "name": "votes", "$ref": "Optional.Integer" },
```

Comment:
These changes to JSON-RPC are not acceptable because they break backwards compatibility and the `votes` property isn't that important that it warrants doing that.